pkg/sloop/webserver/webfiles/debug.html

```diff
@@ -6,14 +6,19 @@
 -->
 <html>
 <head>
-    <script type="text/javascript">
-        document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
+    <!-- Load the base URL and sets Home button reference seperately -->
+    <script>
+        var homeRef = "/" + window.location.pathname.split('/')[1]; 
+        function loadHomeRef() {
```

Comment:
Since this func and homeRef is common would it be better to define it once and reference ?

Reply:
+1
**Suggestion**: Move it to a separate js file and import it this all pages.